archive/import.go

```diff
@@ -132,7 +131,6 @@ func (d *importDriver) close() error {
 		return err
 	}
 	d.dataFileWriter = nil
-
```

Comment:
I can't comment below but with these changes can't we get rid of the `sort` proc we use for import now? In fact can't we just not use the `driver` and make the `importDriver` into a regular ole' `zbuf.Reader`?

Reply:
We need the records in each chunk file to be sorted according to the archive's sort direction. Perhaps you're wondering if we could just take N records at a time from a reader, sort them, and write them, instead of sorting the entire input? If so, we'd potentially be writing overlapping chunks that we wouldn't have if we sorted all of the input at once.

Reply:
> Perhaps you're wondering if we could just take N records at a time from a reader, sort them, and write them, instead of sorting the entire input?

Yes this exactly.

 > If so, we'd potentially be writing overlapping chunks that we wouldn't have if we sorted all of the input at once.

Sure but we wouldn't have to incur the cost of writing the entire stream onto some temp location on disk, then read and write them back to their final destination. Ingest would be a lot faster- it's the beauty of lsm. 

Also, if we go this route, I cannot see how we would allow query during ingest for brim pcap post. I suppose we could arbitrarily break up the stream and do a new import, but in the end this would be the same thing as what I'm suggesting.

Reply:
Let's discuss this on #1132 ; this is good topic, but I don't think we're going to want to add more changes to this PR.